ekncontent/ekncontent/eknc-query-object.c

```diff
@@ -607,6 +622,53 @@ get_query_clause (EkncQueryObject *self)
 }
 
 static gchar *
+get_query_clause2 (EkncQueryObject *self)
```

Comment:
The code duplication isn't ideal, but my thinking was that `get_query_clause()` is likely to be left alone now, while the code in `get_query_clause2()` is likely to evolve further, and so diverge more from `get_query_clause()`. Keeping the code paths for an old bridge and new bridge separate makes it easier to evolve the code for a new bridge without risk of breaking handling of an old bridge.